scheduler/scheduler.go

```diff
@@ -321,6 +322,28 @@ func (s *eremeticScheduler) ScheduleTask(request eremetic.Request) (string, erro
 	}
 }
 
+func (s *eremeticScheduler) Kill(taskId string) error {
+	task, err := s.database.ReadTask(taskId)
+	if err != nil {
+		return err
+	}
+
+	if task.IsTerminated() {
+		return nil
+	}
+
+	_, err = s.driver.KillTask(&mesos.TaskID{Value: proto.String(taskId)})
```

Comment:
This would try to kill tasks even if they are not yet running (and likely fail) and it would then later become running when we get to it in the queue.

So it's not actually giving us a way to clear tasks from the queue that we are not able to launch due to bad resource specs like we wanted :sake: